clj-refactor.el

```diff
@@ -2211,6 +2217,7 @@ form."
   (-when-let (aliases (and cljr-magic-requires
                            (not (cider-in-comment-p))
                            (not (cider-in-string-p))
+                           (not (cljr--in-keyword-p))
```

Comment:
As @Malabarba pointed out, this would now do the wrong thing if you write `::some-alias/some-keyword`